### PR TITLE
Fix 1471 categorical vs categorised

### PR DESCRIPTION
**Pull request by ismailsunni**
To make them match with the categorical vs categorised.

I only change the metadata and the related information to match with how the IF's works. Perhaps this is a short time solution, before we agree `to merge or not to merge` them

cc @akbargumbira @timlinux @Charlotte-Morgan

### CODE REVIEW

File: safe/impact_functions/generic/categorical_hazard_population.py [hunk 1]
```diff
@@ -120,27 +120,25 @@ def get_metadata():
...
+        'Map of population exposed to high category and a table with number '
+        'of people in each category')
     detailed_description = tr(
-        'This function will calculate how many people will be impacted '
-        'per each category for all categories in the hazard layer. '
-        'Currently there should be 3 categories in the hazard layer. After '
-        'that it will show the result and the total amount of people that '
-        'will be impacted for the hazard given.')
+        'This function will calculate how many people will be impacted per '
+        'each category for all categories in the hazard layer. Currently '
```
Comment (timlinux) [inline safe/impact_functions/generic/categorical_hazard_population.py]: Remove the word 'each' here

Comment (timlinux) [inline safe/impact_functions/generic/categorical_hazard_population.py]: Remove Currently and capitalise 'there' in the next line

Comment (ismailsunni) [inline safe/impact_functions/generic/categorical_hazard_population.py]: Fixed in the next commit. Thanks @timlinux

**Comment by cchristelis** [comment]
LGTM

File: safe/impact_functions/generic/classified_hazard_building.py [hunk 2]
```diff
@@ -106,54 +106,55 @@ def get_metadata():
...
+        'hazard. There should be 3 classes: e.g. 1, 2, and 3.')
     exposure_input = tr(
-        'Vector polygon layer which can be extracted from OSM '
-        'where each polygon represents the footprint of a building.')
+        'Vector polygon layer which can be extracted from OSM where each '
+        'polygon represents the footprint of a building.')
     output = tr(
-        'Map of structure exposed to high category and a table with '
-        'number of structure in each category')
+        'Map of structure exposed to high class and a table with number of '
```
Comment (timlinux) [inline safe/impact_functions/generic/classified_hazard_building.py]: The impact layer will contain all structures that were exposed to the highest class (3) and a summary table containing the number of structures in each class.

**Pull request merged**